hs_explore/management/commands/lda_recommend_resources.py

```diff
@@ -0,0 +1,342 @@
+from collections import defaultdict
+from datetime import timedelta, date, datetime
+from hs_tracking.models import Variable
+from hs_core.models import BaseResource
+from hs_core.hydroshare.utils import user_from_id, get_resource_by_shortkey
+from django.core.management.base import BaseCommand
+from hs_explore.models import ResourcePreferences, UserPreferences, OwnershipPreferences
+from hs_explore.models import RecommendedResource, RecommendedUser, RecommendedGroup, \
+    GroupPreferences, PropensityPrefToPair, PropensityPreferences, \
+    UserInteractedResources, UserNeighbors
+import string
+import gensim
+from gensim import corpora
+from nltk.stem import WordNetLemmatizer
+from operator import itemgetter
+from hs_explore.models import LDAWord
+from haystack.query import SearchQuerySet, SQ
+
+
+def get_resource_to_subjects():
```

Comment:
block comment, explanation, etc. Not part of the main code, so that in principle unit tests are not required, but in my opinion they would be a great idea. Not sure that these functions should not be located in hs_explore/utils.py